Rakefile

```diff
@@ -144,6 +221,30 @@ namespace :release do
     exit 2 unless (STDIN.gets.chomp == 'Y')
   end
 
+  desc 'Updates metadata for the new release'
+  task :update_metadata do
+    print "New version of Sourcery in sematic format [x.y.z]? "
+    new_version = STDIN.gets.chomp
+    unless new_version =~ /^\d+\.\d+\.\d+$/ then
+      print "Please set version following the semantic format http://semver.org/\n"
+      exit 3
+    end
+
+    print_info "Updating metadata for #{new_version} release\n"
+
+    # Replace master with the new release version in CHANGELOG.md
+    system(%Q{sed -i '' -e 's/## Master/## #{new_version}/' CHANGELOG.md})
+
+    # Update podspec version
+    podspec_update_version(new_version)
+
+    # Update project version
+    project_update_version(new_version)
+
+    print "Now review and type [Y/n] to commit or cancel the changes. "
+    manual_commit(["CHANGELOG.md", "Sourcery.podspec", "Sourcery.xcodeproj/project.pbxproj"], "docs: update metadata for #{new_version} release")
```

Comment:
Should we also create a new tag and push it? I guess otherwise uploading release and pushing to CP trunk will not work.